src/Switch/Switch.tsx

```diff
@@ -7,18 +7,23 @@ type Props = {
   currentValue: boolean;
   small: boolean;
   onChange: () => void;
+  onText: string;
+  offText: string;
 };
 
-export const Switch: React.FC<Props> = ({ disabled, id, currentValue, small, onChange }) => {
+export const Switch: React.FC<Props> = ({ disabled, id, currentValue, small, onChange, onText, offText }) => {
   return (
     <SwitchParent small={small}>
       <Checkbox type="checkbox" id={id} checked={currentValue} onChange={onChange} disabled={disabled} />
-      {id && (
-        <Label htmlFor={id}>
-          <ToggleInner data-yes="yes" data-no="no" small={small} />
-          <Toggle disabled={disabled} small={small} />
-        </Label>
-      )}
+      <Label htmlFor={id}>
+        <ToggleInner data-yes={onText} data-no={offText} small={small} />
+        <Toggle disabled={disabled} small={small} />
+      </Label>
     </SwitchParent>
   );
 };
+
+Switch.defaultProps = {
+  onText: 'yes',
+  offText: 'no',
+} as Partial<Props>;
```

Comment:
Is this needed? Can we use optional props and give them a default value when being passed into the `Switch` function - `({ ..., onText = 'on', offText = 'off' })`?

Reply:
Apparently I can, I guess the thing I was reading might have been using class components